flair/datasets/entity_linking.py

```diff
@@ -1312,7 +1312,7 @@ def __init__(
                 sample_missing_splits=sample_missing_splits_in_each_corpus,
             )
             corpora.append(corpus)
-        print("...done!")
+        log.info("...done!")
 
```

Comment:
This message could be improved upon, since the content of "what has been done" is context-depended.
My suggestion would be to write 
```
log.info("Done with transforming data into column format and creating corpora...")
```